CHANGELOG/CHANGELOG-1.4.md

```diff
@@ -24,6 +24,7 @@ See [code changes](https://github.com/aws/aws-k8s-tester/compare/v1.3.9...v1.4.0
 - Clean up [`colorstring` printf](https://github.com/aws/aws-k8s-tester/pull/101).
 - Clean up [polling operation error handling](https://github.com/aws/aws-k8s-tester/commit/26627f14f4dbbcc8dd64d6307ed6e58c0b809f52).
   - Rename [`eks/cluster/version-upgrade.Poll` to `eks/cluster/wait.PollUpdate`](https://github.com/aws/aws-k8s-tester/commit/a6eeea26a7ab3c7069a4278026b56de87707c9b1).
+- [`eks/mng/scale` added to scale mngs up and down](https://docs.aws.amazon.com/cli/latest/reference/eks/update-nodegroup-config.html)
```

Comment:
Link this PR https://github.com/aws/aws-k8s-tester/pull/106, and add another line for

```
  - See https://docs.aws.amazon.com/cli/latest/reference/eks/update-nodegroup-config.html.
```